README.md

```diff
@@ -10,48 +10,44 @@ The core module for performing simulations. See the [online documentation](http:
 ## Building from source
 
 ### Linux / macOS
-The build is tested on Ubuntu 18.04 with gcc-9.1 / clang-8 and on macOS Mojave
-10.14.5 with dependencies from Homebrew and Apple LLVM 10.
+The build is tested on Ubuntu 18.04 with gcc-8 / clang-8 and on macOS Mojave
+10.14.5 and apple clang.
 
 #### Requirements
-On Linux you will need a C++17 capable compiler and a standard library that
-supports `<filesystem>`. On macOS you will need the system compiler with C++17
-and filesystem from boost.
+For the compilation of JuPedSim core you need a C++17 capable compiler and standard library.
```

Comment:
For the compilation of `jpscore` you need a C++17 capable compiler and standard library.